src/main/java/games/strategy/triplea/delegate/BattleCalculator.java

```diff
@@ -461,8 +463,12 @@ public static CasualtyDetails selectCasualties(final String step, final PlayerID
     }
     final GameData data = bridge.getData();
     final boolean isEditMode = BaseEditDelegate.getEditMode(data);
-    final ITripleAPlayer tripleaPlayer = (ITripleAPlayer) bridge.getRemotePlayer(player);
-
+    final ITripleAPlayer tripleaPlayer;
+    if (player.isNull()) {
+      tripleaPlayer = new WeakAI(player.getName(), TripleA.WEAK_COMPUTER_PLAYER_TYPE);
+    } else {
+      tripleaPlayer = (ITripleAPlayer) bridge.getRemotePlayer(player);
+    }
```

Comment:
Would a ternary operator fit within the 120 char limit?

Reply:
No, but, even with wrapping, it reduces the line count by 50%. :smile:  Done in rebased eca7723.